server/neptune/terraform/async_client.go

```diff
@@ -98,6 +114,9 @@ func (c *AsyncClient) runCommand(ctx context.Context, jobID string, path string,
 	// Ensure we close our channels when we exit.
 	defer func() {
 		close(outCh)
+
+		// Close Job and Persist to storage
+		c.JobOutputHandler.CloseJob(ctx, jobID)
```

Comment:
can this happen one level above? We have commands like `terraform show` that we won't want to forward logs through for.  imo it should be the callers decision to do things with the output.

Reply:
The way to recognize this, is like as I'm typing out a call to this client, I'm wondering why this should care about job id and then i have to go in and see that there's this dependency.